### ISSUE TITLE
CLI-1131: Can not login after Cloud Platform changes on 

### ISSUE THREAD

**Issue by korkhau** [acquia/cli#1582]
**Describe the bug**
It looks after change https://docs.acquia.com/cloud-platform/release-notes/#cloud-platform-august-20-2023-through-august-26-2023 ACLI is no longer has a possibility to login as it requires additional auth from https://docs.acquia.com/cloud-platform/develop/api/auth/#making-api-calls-through-single-sign-on

**To Reproduce**
1. Run `acli auth:login`
2. Try to retrieve some from API env specific like backups list `acli api:environments:database-backup-list {uuid} {db} --limit 1 --sort desc -vvv`
3. Observe the error:
```
{
    "error": "additional_authentication_required",
    "message": "This resource requires additional authentication."
}
```

**Expected behaviour**
Acquia CLI returns some backups 

**Additional context**
Reproducible for the latest version of Acquia CLI as well as for 2.9.5

**Comment by anavarre** [comment]
We have released a fix in Acquia CLI 2.14.0 last week. If you're not using Acquia CLI in the context of Cloud IDE, you'll have to set an environment variable yourself:

```
export AH_ORGANIZATION_UUID=<org-uuid>
```

**Comment by korkhau** [comment]
@anavarre,

We've restarted a container several times so this is looks like not an issue due to cached creds.

We found that disabling Federated Login fixes the issue. 
So we assume this is a root cause as per investigations:
1) We request GET /api/environments/{env}/databases/{db}/backups?sort=desc&limit=1
2) We see in verbose response next: 
```
HTTP/1.1 403 Forbidden
< Date: Mon, 18 Sep 2023 15:38:23 GMT
...
< X-CloudAPI-SSO: active; url={vendor_host}/am/SSORedirect/metaAlias/alpha/idp?SAMLRequest={request_id}; type=saml
```
After login via Web UI with the user we use client_id + client_secret we successfully logged in via acli.

Thanks
